AZ3166/jenkins/deployment/DevKitDeploy.ps1

```diff
@@ -39,6 +39,16 @@ foreach($StorageAccountName in $StorageHashTable.Keys)
 	$ArduinoPackageBlobName = "AZ3166-" + $CurrentVersion + ".zip"
 	Set-AzureStorageBlobContent -Context $StorageContext -Container $Environment -File $ArduinoPackageFilePath -Blob "$ArduinoPackageContainer\$ArduinoPackageBlobName" -Force
 
+	# Upload Firmware bin file
+	$FirmwareFileName = "devkit-firmware-" + $CurrentVersion + "." + $env:BUILD_NUMBER + ".bin"
+	$FirmwareFilePath = Join-Path -Path (Get-Location).Path -ChildPath "TestResult\$FirmwareFileName"
+	Set-AzureStorageBlobContent -Context $StorageContext -Container $Environment -File $FirmwareFilePath -Blob $FirmwareFileName -Force
+
+	# Upload Firmware bin file for OTA
+	$OTAFirmwareFileName = "devkit-firmware-latest.ota.bin"
```

Comment:
version

Reply:
For OTA firmware, there is no version

Reply:
No version here. It will replace old firmware directly so that OTA sample can always get latest firmware with url.

Reply:
I see, it's for the OTA sample